### PR TITLE
Add advertisement time sheets

### PR DESCRIPTION
**Pull request by EvalVis**
Integrating time picker in add advertisement.

### CODE REVIEW

**Comment by Yar3k** [comment]
I would remake time picker.

**Review by Yar3k — COMMENTED** [review]
Like this

**Comment by EvalVis** [comment]
This seems better. However, try to google how to set 24 hours format.

File: Support_Your_Locals/Views/Shared/TimePicker.cshtml [hunk 1]
```diff
@@ -7,42 +7,11 @@
     string pickerTo = "picker" + day7Index;
```
Comment (EvalVis) [inline Support_Your_Locals/Views/Shared/TimePicker.cshtml:7]: You do not need @{
} then.

File: Support_Your_Locals/Views/Shared/TimePicker.cshtml [hunk 2]
```diff
@@ -7,42 +7,11 @@
...
 }
 
-<link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" />
-<link href="/lib/datetimepicker/css/jquery.datetimepicker.min.css" rel="stylesheet" />
-
 <div class="container mt-5 mb-5" style="width: 400px">
     <label>@TimeSheetRegisterViewModel.Days[dayIndex]</label>
     <input type="hidden" asp-for="Weekday" value="@(dayIndex + 1)" />
```
Comment (EvalVis) [inline Support_Your_Locals/Views/Shared/TimePicker.cshtml:16]: Just call ViewBag.DayInd + 1

**Comment by EvalVis** [comment]
@Yar3k if you do not mind, I reverted "remake timepicker" because that does not work on Firefox, which is widely used.
Do you agree, with this revert?

**Review by Yar3k — APPROVED** [review]
Approving

**Review by MartynaUb — APPROVED** [review]
All looks good.

**Pull request merged**